### PR TITLE
gnugpg call now use explicitly gpg1 binary

### PR DESCRIPTION
**Pull request by M0Rf30**
gpg1 binary is used in every distro. I think it's the fastest fix you could apply. I've just updated my non-git PKGBUILD in archlinux aur repo with a patch and it works granting you a wide testing ground by archlinux users. more users...more fixes :dancers:

### CODE REVIEW

**Comment by hoffmabc** [comment]
My only question is whether or not gpg1 will always be in /usr/bin. This should probably be configurable.

**Comment by M0Rf30** [comment]
Could you use a which command for gpg1 in order to retrieve the correct path dynamically?

**Comment by M0Rf30** [comment]
twisted has is which implementation and is just an used library for this project

**Comment by M0Rf30** [comment]
from the python-gnupg docs:
Name for GnuPG binary executable. If the absolute path is not given, the evironment variable $PATH is searched for the executable and checked that the real uid/gid of the user has sufficient permissions.

File: node/transport.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def _setup_settings(self):
             try:
                 self.log.info('Generating PGP keypair. This may take several minutes...')
                 print 'Generating PGP keypair. This may take several minutes...'
-                gpg = gnupg.GPG()
+                gpg = gnupg.GPG(gpgbinary='gpg1')
```
Comment (gubatron) [inline node/transport.py]: what about windows?

**Comment by M0Rf30** [comment]
fixed with latest python2-gnupg commits

**Comment by Renelvon** [comment]
@M0Rf30 If the issue is fixed and the patch is no longer needed, please close this pull request.

**Pull request closed without merging**